sdk/eventhub/azure-eventhub/tests/livetest/asynctests/test_send_async.py

```diff
@@ -155,15 +156,38 @@ async def test_send_with_partition_key_async(connstr_receivers):
         await client.send_batch(await client.create_batch())
 
     found_partition_keys = {}
+    reconnect_receivers = []
     for index, partition in enumerate(receivers):
-        received = partition.receive_message_batch(timeout=5000)
-        for message in received:
+        retry_total = 0
+        while retry_total < 3:
+            timeout = 5000 + retry_total * 1000
```

Comment:
@yunhaoling - Roundtrips were around 3 minutes from East US to Australia (which should be longest distance/roundtrip from point to point) when this test was run locally. Might be longer in the pipeline, so setting to five minutes to start with, then bumping up.

Reply:
I'm not sure whether `5 minutes total round trip` is related to the code here: `timeout = 5000 + retry_total * 1000`,

but I just want to call it out that `receive_message_batch` takes timeout in the unit of milliseconds.

`setting 5 minutes` should be 60 * 5 * 1000 = 300000 then?

Reply:
that makes more sense :D

Reply:
This was the timeout that worked locally (6/7/8 secs not 6/7/8 mins :P). So setting timeout to less than `5000 + retry_total * 1000` resulted in the UnknownError.

Reply:
yeah, an estimation of 3 minutes in total for connection to all partitions seems reasonable..
okay, now I know what this time unit it is and I'm good with the change.